test_coverage/schema_validation_testing.py

```diff
@@ -3,34 +3,26 @@
 from data_generators.data_options import generate_uuid
 
 
-
 def test_mentor():
-    mentor = RandomMentor()
-    mentor_dict = vars(mentor)
-
-    assert Mentor(**mentor_dict)
+    assert all(Mentor(**vars(RandomMentor()) for _ in range(1000)))
 
 
 def test_mentee():
     mentee = RandomMentee()
-    mentee_dict = vars(mentee)
 
-    assert Mentee(**mentee_dict)
+    assert all(Mentee(**vars(mentee) for _ in range(1000)))
```

Comment:
This tests the same RandomMentee instance 1000 times. The RandomMentor  above is correct. Check meeting and feedback.

Reply:
Just seeing this, looking into it presently